### DISCUSSION TITLE
Nouvelle mécanique de jeu : l'accouplement (ou l'évolution) des pets

### DISCUSSION THREAD

**Discussion by Procion06** [Crownicles/Crownicles#2138] [Nouvelles idées]
Afin de redynamiser complètement ce secteur du jeu sous exploité et d'attirer de nouveaux joueurs, je propose de faire évoluer nos pets !
cette idée rejoint et regroupe certains points de vue déjà exprimés

le principe est simple et facile à coder 
un joueur possède un pet femelle et un autre joueur un pet mâle de même race
les 2 joueurs lancent un trade (accouplement) et moyennant un prix qui dépend du nombre d'étoiles des pets, il en résulte un pet de même race avec une étoile en plus (le mâle et la femelle sont supprimés)
c'est le joueur qui possède le pet femelle qui paie et obtient le nouveau pet

un exemple :

je possède une louve 2*
un joueur possède un loup 2*

je fais la commande d'accouplement et je paie le prix correspondant au nouveau pet que je vais obtenir, ici 10000
les 2 pets (le loup et la louve) disparaissent
et j'obtiens un loup 3* (aléatoirement mal ou femelle)

exemple de prix (à définir) :

pour passer de 1 à 2* : 5000
pour passer de 2 à 3* : 10000
pour passer de 3 à 4* : 20000
pour passer de 4 à 5* : 35000

je pense que ça permettra de générer un échange croissant entre les joueurs, notamment ceux qui aiment cette mécanique de jeu ou tout simplement ceux qui souhaitent voir leur animal préféré "monter en puissance"
qui n'a jamais rêvé de traverser le Royaume avec son animal de compagnie, un chien un chat ou encore son animal totem ?
plutôt que de se trimballer avec un alien qui peut vous enlever n'importe quand ou un dragon qui peut vous cramer s'il est un peu trop énervé ^^

cette proposition comporte un attrait ludique indéniable pour les joueurs et relancerait complètement l'utilité et le "trade" de pets dans Draftbot :)
on peut aussi décider que le nouveau pet restera un bébé pendant une durée déterminée (par ex 7 jours) avant de devenir adulte

Tous les paramètres sont modifiables bien sûr sauf l'idée de base, pouvoir faire évoluer n'importe quel pet du jeu !

**Comment by Nargacaura** [comment]
L'idée de l'accouplement des pets avait déjà été proposé dans la discussion #621.
De plus, à mon avis (donc l'avis des contribs peut être différent du mien), ça sera vraiment complexe à implémenter pour ce que ça aura potentiellement comme valeur au niveau du gameplay (j'suis pas sûr que c'est énormément demandé, ça...).

**Comment by Procion06** [comment]
la discussion à laquelle tu fais référence parle de durée de vie et de vieillissement des pets
ici c'est très simple, tu fusionnes 2 pets et tu paies pour en avoir un identique mais avec une étoile en plus
concernant l'implémentation, c'est le même principe qu'un trade de pets entre 2 joueurs, et d'une vente, donc il y a déjà une base
